portage.Dockerfile

```diff
@@ -11,18 +15,28 @@ ARG SNAPSHOT="portage-latest.tar.xz"
 ARG DIST="https://ftp-osl.osuosl.org/pub/gentoo/snapshots"
 ARG SIGNING_KEY="0xEC590EEAC9189250"
 
-RUN apk add --no-cache ca-certificates gnupg tar wget xz \
- && wget -q "${DIST}/${SNAPSHOT}" "${DIST}/${SNAPSHOT}.gpgsig" "${DIST}/${SNAPSHOT}.md5sum" \
- && gpg --list-keys \
- && echo "honor-http-proxy" >> ~/.gnupg/dirmngr.conf \
- && echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf \
- && gpg --keyserver hkps://keys.gentoo.org --recv-keys ${SIGNING_KEY} \
- && gpg --verify "${SNAPSHOT}.gpgsig" "${SNAPSHOT}" \
- && md5sum -c ${SNAPSHOT}.md5sum \
- && mkdir -p var/db/repos var/cache/binpkgs var/cache/distfiles \
- && tar xJpf ${SNAPSHOT} -C var/db/repos \
- && mv var/db/repos/portage var/db/repos/gentoo \
- && rm ${SNAPSHOT} ${SNAPSHOT}.gpgsig ${SNAPSHOT}.md5sum
+RUN <<-EOF
+    set -e
+
+    apk add --no-cache ca-certificates gnupg tar wget xz
+    wget -q "${DIST}/${SNAPSHOT}" "${DIST}/${SNAPSHOT}.gpgsig" "${DIST}/${SNAPSHOT}.md5sum"
+
+    # setup GPG
+    gpg --list-keys
+    # following MUST be without intendation because alpine's shell doesn't support <<-EOF syntax
+    cat <<GPG >> ~/.gnupg/dirmngr.conf
```

Comment:
If you do `<<-GPG` here, you can indent the `honor-http-proxy`, `disable-ipv6`, and `GPG` lines.

Reply:
We're using alpine's image here, so need to follow busybox sh's rules

Reply:
meh, ok

Reply:
It does support it. My testing method was wrong. Lmao. I was using 4 spaces instead of <tab>.

Reply:
👀